package.json

```diff
@@ -33,6 +34,7 @@
   "devDependencies": {
     "@babel/cli": "^7.0.0",
     "@babel/core": "^7.0.1",
+    "@babel/plugin-transform-runtime": "^7.0.0",
```

Comment:
This will require end-users to have babel runtime installed as well, right? I thought that it wasn't good practice to use this in an npm package

Reply:
This is actually is the best practice since it allows to share the same helpers with other packages. And if user also have babel then his helpers will be also reused.

Reply:
Done